src/js/subtraction/components/Attribution.tsx

```diff
@@ -3,6 +3,11 @@ import styled from "styled-components";
 import { getColor, getFontSize, theme } from "../../app/theme";
 import { Attribution, AttributionWithName } from "../../base";
 
+/**
+ * Styled component for the attribution when none found.
```

Comment:
```suggestion
 * Default attribution used when creation details are unknown
```
Changes: 

1. Mentioning the usage of a styled component is redundant since it should be obvious
2. `when none found` is strange wording.  What is it trying to convey?